api/nakadi-event-bus-api.yaml

```diff
@@ -904,6 +1187,23 @@ definitions:
         description: |
           Offset of the event being pointed to.
 
+  CursorCommitResult:
+    description: |
+      The result of single cursor commit. Holds a cursor itself and a result value.
+    required:
+      - cursor
+      - result
+    properties:
+      cursor:
+        $ref: '#/definitions/Cursor'
+      result:
+        type: string
+        description: |
+          The result of cursor commit.
+          - `committed`: cursor was successfully committed
+          - `outdated`: there already was more recent (or the same) cursor committed, so the current one was not
+            committed as it is outdated
+
   EventStreamBatch:
```

Comment:
That's a good idea. We added a new object, but did not completely defined it's structure now, in order to not make this discussion even longer https://github.com/zalando/nakadi/pull/353/commits/b68f324ed41db9489c40f8c2d43b8507aae89db8